README.md

```diff
@@ -19,23 +19,19 @@ concurrent language. This document describes how to build `juju` from source.
 If you are looking for binary releases of `juju`, they are available in the snap store
 
     snap install juju --classic
-    
+
 Installing Go
 --------------
 
-`Juju's` source code currently depends on Go 1.9. One of the easiest ways
+`Juju's` source code currently depends on Go 1.10. One of the easiest ways
 to install golang is from a snap. You may need to first install
 the [snap client](https://snapcraft.io/docs/core/install). Installing the golang
 snap package is then as easy as
 
-    snap install go --classic
+    snap install go --channel=1.10/stable --classic
 
 You can read about the "classic" confinement policy [here](https://insights.ubuntu.com/2017/01/09/how-to-snap-introducing-classic-confinement/)
 
-If you want to use `apt`, then you can add the [juju-golang PPA](https://launchpad.net/~juju/+archive/ubuntu/golang) and then run the following
```

Comment:
Let's retain this information, but change the ppa to https://launchpad.net/~gophers/+archive/ubuntu/archive

Reply:
Thanks Kelvin! I'm sorry, I didn't tell you that ppa name was also different. Could you update the name to "Golang Gophers PPA" instead of "juju-golang PPA"?

Reply:
ah, sorry, @nskaggs I should have updated this. Thanks for reminding me.